powdr_cli/src/main.rs

```diff
@@ -82,6 +87,19 @@ enum Commands {
         force: bool,
     },
 
+    /// Apply the Halo2 workflow on an input file and prover values.
```

Comment:
In addition to that, shouldn't we also have an option to create the proof for all the other subcommands that perform witness generation?

Reply:
#301